doc/fixing-broken-sites.md

```diff
@@ -0,0 +1,39 @@
+# How to fix broken site issues
+
+Unfortunately, while working to protect your privacy, Privacy Badger can end up breaking website functionality. This document is about the process of classifying and resolving these breakages.
+
+Here are the [open "broken site" and "good-volunteer-task"-labeled issues](https://github.com/EFForg/privacybadger/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22broken%20site%22%20label%3A%22good%20volunteer%20task%22).
+
+The first thing to do is to confirm that Privacy Badger blocks (or will eventually learn to block) the domain, and that blocking the domain does indeed break the site.
```

Comment:
I think many user's who come to this document will already have a broken site, and they will be looking for how to figure out *what* broke the site. So I think it would be very useful to describe a simple debugging processes using the popup's sliders. Maybe @terrorist96 would have good advice here. 

From there we could potentially describe how to further investigate the trackers that break the site by  opening devtools and looking for cookies in the network requests.

However this document is an improvement even without this information, so feel free to merge it. We can always add more later.